index.bs

```diff
@@ -88,7 +88,7 @@ The use of <strong>CSS layout methods</strong> such as <strong>Flexbox</strong>
 The use of <strong>iframes</strong> produces the closest effect to what Container Queries is intended to achieve, but their use has two significant drawbacks:
 <ol>
 <li>
-Without the use of JavaScript, iframes require that the contents of the iframe are stored in a <em>separate HTML document</em> rather than being part of the current document. This necessitates a new document composition paradigm that’s very different from the methods currently employed by developers.
+Without the use of JavaScript, some browsers require that the contents of the iframe are stored in a <em>separate HTML document</em> rather than being part of the current document. This necessitates a new document composition paradigm that’s very different from the methods currently employed by developers.
 </li>
 <li>
 The contents of an iframe do not natively affect the dimensions of the iframe itself. The contents of a container that is the subject of a Container Query are still expected to be able to affect the layout of its parents in the <em>opposite</em> dimension of the query. For example, in the case of a Container Query that is based on a containers <em>width</em>, it is still expected that the container’s contents affect the <em>height</em> of a container, subject to any overflow rules.
```

Comment:
Despite having some security implications, iframes on mobile safari on iOS _are_ affected by the document contained inside them. By default, the height of the iframe expands to match the height of the embedded document.

Reply:
^ this was referring to this wording:

> The contents of an iframe do not natively affect the dimensions of the iframe itself.

Reply:
Ok, thanks. Will consider how to re-phrase that.

Reply:
@tomhodgins - I've re-phrased. This part of the conversation is now 'outdated'.